pkg/oc/bootstrap/docker/openshift/templateservicebroker.go

```diff
@@ -76,5 +70,65 @@ func (h *Helper) InstallTemplateServiceBroker(f *clientcmd.Factory, imageFormat
 		return errors.NewError(fmt.Sprintf("failed to start the template service broker apiserver: %v", err))
 	}
 
+	// Register the template broker with the service catalog
+	glog.V(2).Infof("registering the template broker with the service catalog")
```

Comment:
followup issue to make this a template somewhere please

Reply:
done: https://github.com/openshift/origin/issues/16075